packages/rag/question_generator.py

```
@@ -0,0 +1,358 @@
+        "topP": 0.8,
+        "topK": 0,
+        "maxCompletionTokens": max_tokens,
+        "temperature": temperature,
+        "repetitionPenalty": 1.1,
+        "thinking": {"effort": "high"},
+        "stop": [],
+        # responseFormat 제거됨 (Reasoning 모델 사용 시 프롬프트 지시로 대체)
+    }
+
+    # API 호출
+    response = requests.post(url, headers=headers, json=data)
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)